libnavui-maneuver/src/main/res/drawable/mapbox_ic_merge_slight_left.xml

```diff
@@ -1,14 +1,39 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="UTF-8"?><!-- Generated by PaintCode - http://www.paintcodeapp.com -->
+<!-- merge (flipped) -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="20dp"
-    android:height="20dp"
-    android:viewportWidth="20"
-    android:viewportHeight="20">
+    xmlns:aapt="http://schemas.android.com/aapt"
+    android:name="merge"
```

Comment:
```suggestion
    android:name="mergeSlightLeft"
```

Reply:
@abhishek1508 Are we using those names for anything?

Reply:
I've only seen `android:name` used on a `<group>` and `<path>` tags. Never in `<vector>`.  They were probably added by PaintCode during export.